src/fieri/.env.test

```diff
@@ -0,0 +1 @@
+REDIS_URL='redis://localhost:6379/2/fieri'
```

Comment:
Added this to fix weird race-ish conditions when running the tests while also having a development instances of Supermarket and it's Sidekiq workers up. Gives a Fieri test run a kinda dedicated redis database separate from Supermarket's default.